projects/optic/src/commands/diff/generate-rule-runner.ts

```diff
@@ -1,31 +1,59 @@
 import path from 'path';
 import { OpticCliConfig } from '../../config';
 import { StandardRulesets } from '@useoptic/standard-rulesets';
-import { RuleRunner, Ruleset, prepareRulesets, ExternalRule } from '@useoptic/rulesets-base';
-import { createOpticClient } from '@useoptic/optic-ci/build/cli/clients/optic-client';
+import {
+  RuleRunner,
+  Ruleset,
+  prepareRulesets,
+  ExternalRule,
+} from '@useoptic/rulesets-base';
+import { loadCliConfig } from '../../config';
 
 const isLocalJsFile = (name: string) => name.endsWith('.js');
 
 type InputPayload = Parameters<typeof prepareRulesets>[0];
 
+const getStandardToUse = async (
+  options: {
+    specRuleset?: string;
+    rulesetArg?: string;
+    config: OpticCliConfig;
+  }
+): Promise<OpticCliConfig['ruleset']> => {
+  // We always take the --ruleset arg as priority, then the ruleset on the API spec (from the head), then fallback to the optic.dev.yml config
```

Comment:
> then fallback to the optic.dev.yml config

Aidan mentioned wanting to get rid of optic.dev.yml. Is that going forward?

Reply:
I don't see a reason we need to - I figure we can keep around this and use it as a fallback for an entire repo config (which also has a default fallback of breaking-changes on)

What's the reason we'd want to get rid of it (cc @acunniffe)